hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_matcher.py

```diff
@@ -88,11 +112,19 @@ def lambda_handler(event, context):
                 ).write_to_table(records_table)
 
                 match_ids.append(signal_id)
-            sns_client.publish(
-                TopicArn=OUTPUT_TOPIC_ARN,
-                Subject="Match found in pdq_matcher lambda",
-                Message=f"Match found for key: {key}, hash: {hash_str}, for IDs: {match_ids}",
-            )
+                message = MatchMessage(
+                    content_key=key,
+                    content_hash=hash_str,
+                    banked_indicator_id=signal_id
```

Comment:
Would expect the default labels to be added here. You can also add a TODO to add labels from the ThreatExchange data, if that happens here (it should happen wherever the bank metadata is being looked up).